content/en/mpas/getting_started.md

```diff
@@ -42,11 +42,12 @@ For other installation methods, see the [installation guide](/mpas/overview/inst
 ### Export your GitHub access token
 
 The MPAS CLI uses your GitHub access token to authenticate with GitHub. To create a
-GitHub access token, see the [GitHub documentation](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+GitHub access token, see the [GitHub documentation](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token). In addition to that we need to export your GitHub user and an your email address as they are used later.
 
 ```bash
 export GITHUB_TOKEN=<your-github-access-token>
 export GITHUB_USER=<your-username>
+export MY_EMAIL=<your-email-address>
```

Comment:
The email is optional actually.

Reply:
maybe, but then one cannot copy the example code snippets...